### ISSUE TITLE
Unable to Update Windows Repo On Masterless Windows Minion

### ISSUE THREAD

**Issue by DWSR** [saltstack/salt#51015]
### Description of Issue/Question
I am unable to follow [these instructions](https://docs.saltstack.com/en/latest/topics/windows/windows-package-manager.html#managing-windows-software-on-a-standalone-windows-minion) for managing the Windows Software repo on a Masterless Winion due to an error when executing `salt-call winrepo.update_git_repos`. A workaround is to call it via `salt-call saltutil.runner name=winrepo.update_get_repos`. Once using the workaround, the second and third commands execute successfully.

### Setup
Default install of 2018.3.3 and default config file except `file_client: local`

### Steps to Reproduce Issue
```
> salt-call -l debug winrepo.update_git_repos
[DEBUG   ] Reading configuration from c:\salt\conf\minion
[DEBUG   ] Using cached minion ID from c:\salt\conf\minion_id: DESKTOP-PIC2LV8.localdomain
[DEBUG   ] Configuration file path: c:\salt\conf\minion
[WARNING ] Insecure logging configuration detected! Sensitive data may be logged.
[DEBUG   ] Grains refresh requested. Refreshing grains.
[DEBUG   ] Reading configuration from c:\salt\conf\minion
[DEBUG   ] Determining pillar cache
[DEBUG   ] LazyLoaded jinja.render
[DEBUG   ] LazyLoaded yaml.render
[DEBUG   ] LazyLoaded jinja.render
[DEBUG   ] LazyLoaded yaml.render
[DEBUG   ] LazyLoaded reg.read_value
[DEBUG   ] LazyLoaded winrepo.update_git_repos
[ERROR   ] Failed to update winrepo_remotes: name 'GLOBAL_ONLY' is not defined
Traceback (most recent call last):
  File "C:\salt\bin\lib\site-packages\salt\runners\winrepo.py", line 222, in update_git_repos
    global_only=GLOBAL_ONLY,
NameError: name 'GLOBAL_ONLY' is not defined
[DEBUG   ] LazyLoaded nested.output
local:
    Failed to update winrepo_remotes: name 'GLOBAL_ONLY' is not defined
```

### Versions Report
```
Salt Version:
           Salt: 2018.3.3

Dependency Versions:
           cffi: 1.10.0
       cherrypy: 10.2.1
       dateutil: 2.6.1
      docker-py: Not Installed
          gitdb: 2.0.5
      gitpython: 2.1.3
          ioflo: Not Installed
         Jinja2: 2.9.6
        libgit2: Not Installed
        libnacl: 1.6.1
       M2Crypto: Not Installed
           Mako: 1.0.6
   msgpack-pure: Not Installed
 msgpack-python: 0.4.8
   mysql-python: Not Installed
      pycparser: 2.17
       pycrypto: 2.6.1
   pycryptodome: Not Installed
         pygit2: Not Installed
         Python: 3.5.3 (v3.5.3:1880cb95a742, Jan 16 2017, 16:02:32) [MSC v.1900 64 bit (AMD64)]
   python-gnupg: 0.4.1
         PyYAML: 3.12
          PyZMQ: 16.0.3
           RAET: Not Installed
          smmap: 2.0.5
        timelib: 0.2.4
        Tornado: 4.5.1
            ZMQ: 4.1.6

System Versions:
           dist:
         locale: cp1252
        machine: AMD64
        release: 10
         system: Windows
        version: 10 10.0.17134 SP0 Multiprocessor Free
```

**Comment by garethgreenaway** [comment]
@twangboy @dwoz FYI

**Comment by twangboy** [comment]
I can replicate this on the v2018.3.3 tag. It seems to be fixed on the head of 2018.3. Will be fixed in 2018.3.4
